03week/airplane/css/style.css

```diff
@@ -0,0 +1,180 @@
+/* You can style your links page here */
+*{
+    padding:0;
+    margin:0;
+}
+body {
+    background-image:url("../img/airline.jpg");
+    background-size: cover;}
+
+    .top {
+    width: 100%;
+    height: 12%;
+    margin: 0px;
+    position: absolute;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-items: stretch;
+    z-index: -1;
+    border-bottom: 1px solid white;
+    background-color: #1b3b53;
+}
+
+.logo{
+    width:50%;
+    height: auto;
+    margin: 0px;
+    padding-left: 5%;
+    position: relative;
+    left: 0%;
+}
+
+
+
+.navbar { 
+    float: right;
+    position: relative;
+    left: 100px;
+}
+
+.navbar a {
+    text-decoration: none;
+    color:white;
+    position: relative;
+    list-style-type: none;
+    display: inline-block;
+    margin-top: 10%;
+    padding-right: 15px;
+
+}
+.home:hover{
+    color: #33997f;
+}
+
+.iconbar {
+    margin: 250px;
+    padding:20px;
+    position: absolute;
+    float: left;
+    left: -105px;
+    top:-15px;
+}
+
+.button{
+    display: inline-block;
+    border: none;
+    background-color: #33997f;
+    width:140px;
+    height:70px;
+    margin-right: -5px;
+    background-size: auto;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-radius: 3px;
+    
+
+}
+.button:hover{
+    background-color: #0066cc;    
+}
+#flight{
+    background-image: url("../img/plane-icon.png")
+}
+#hotel{
+    background-image: url("../img/hotel-icon.png");
+    
+}
+
+#rentalcar {
+    background-image: url("../img/car-icon.png");
+    
+}
+
```

Comment:
Might be a better idea to put the images in html. since those aren't "background image"